iot-hub/Quickstarts/InvokeDeviceMethod/InvokeDeviceMethod.csproj

```diff
@@ -2,11 +2,11 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Devices" Version="1.28.1" />
+    <PackageReference Include="Microsoft.Azure.Devices" Version="1.35.0-preview-001" />
```

Comment:
@drwill-ms I am planning to remove the Quickstarts and Tutorials from the `preview` branch. These samples are the ones that are linked to MS docs, so I don't see the need to maintain a `preview` copy of these. In the event we add a preview specific feature and publish a corresponding MS Doc, we can add it to the `preview` branch.
What do you think?